pleskdistup/common/src/action.py

```diff
@@ -467,6 +472,8 @@ def make_checks(self) -> typing.List[str]:
                     failed_checks_msgs.append(
                         f"Required pre-conversion condition {check.name!r} not met:\n\t{check.description}\n"
                     )
+            except UnicodeDecodeError as e:
```

Comment:
Yep, because I want to be able to identify the exact UnicodeDecodeError exception at the next level.
If I understand correctly, the next clause will replace the type with RuntimeError, concealing the original error inside.

Reply:
Losing `check.name` isn't good. Instead, for example, you can check `RuntimeError.__cause__`.